olm/update-resources.sh

```diff
@@ -222,6 +222,17 @@ updateNighltyBundle() {
 
     # set `app.kubernetes.io/managed-by` label
     yq -riSY  '(.spec.install.spec.deployments[0].spec.template.metadata.labels."app.kubernetes.io/managed-by") = "olm"' "${NEW_CSV}"
+    
+    # set Pod Security Context Posture
+    yq -riSY  '(.spec.install.spec.deployments[0].spec.template.spec."hostIPC") = "false"' "${NEW_CSV}"
+    yq -riSY  '(.spec.install.spec.deployments[0].spec.template.spec."hostNetwork") = "false"' "${NEW_CSV}"
+    yq -riSY  '(.spec.install.spec.deployments[0].spec.template.spec."hostPID") = "false"' "${NEW_CSV}"    
+    yq -riSY  '(.spec.install.spec.deployments[0].spec.template.spec.containers[0].securityContext."privileged") = "false"' "${NEW_CSV}"
+    yq -riSY  '(.spec.install.spec.deployments[0].spec.template.spec.containers[0].securityContext."readOnlyRootFilesystem") = "false"' "${NEW_CSV}"
+    if [ "${platform}" == "openshift" ]; then
+      yq -riSY  '(.spec.install.spec.deployments[0].spec.template.spec.containers[0].securityContext."allowPrivilegeEscalation") = "false"' "${NEW_CSV}"
```

Comment:
```suggestion
      yq -riSY  '(.spec.install.spec.deployments[0].spec.template.spec.containers[0].securityContext."allowPrivilegeEscalation") = false' "${NEW_CSV}"
```

Reply:
Resolved in the latest commit: https://github.com/eclipse-che/che-operator/pull/811/commits/7652785943807cccaef9cb50a12dacae98782516